### DISCUSSION TITLE
Used by counter not showing the packages

### DISCUSSION THREAD

**Discussion by lucabriguglia** [community/community#22955] [New to GitHub]
<p>Hi,</p>
<p>Do I need to do anything in particular in order for my packages to be shown in the select package drop down list?</p>
<p>I have 12 packages in my solution and none are displayed, the are only two listed that are no longer part of the solution.</p>
<p>Thanks,</p>
<p>Luca</p>

**Comment by lucabriguglia** [comment, marked as answer]
<p>Solved by adding a .nuspec file.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/lucabriguglia">@lucabriguglia</a>, thanks for coming back to share what worked for you. Much appreciated!</p>

**Comment by lucabriguglia** [comment]
<p>Thanks <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>, even though I need to specify that it worked only for one package, all the others are still hidden. Mistery, and there is no mention anywhere in the docs on how to set this up.</p>
<p>Cheers,</p>
<p>Luca</p>
